css/src/core/bare-elements.scss

```diff
@@ -32,3 +32,16 @@ fieldset {
 	min-width: 0; // as a replaced element min-width:0 is required to enable it to work in responsive layouts
 	border: none;
 }
+
+::target-text {
+	background-color: $info-background;
```

Comment:
Wondering if `$code-highlight-background` might work as a better color to us, since it's already used for this purpose in code, and might not need to care about `color`. It might also ensure high contrast works a bit better.

```css
::target-text {
	background-color: $code-highlight-background;
}
```

<img width="603" alt="image" src="https://user-images.githubusercontent.com/30843002/231605169-859f1de8-692e-496e-93f4-4e6a080c9109.png">


Don't feel extremely strongly about this, but it might be worth experiementing.